test-framework/test-suites/integration/files/dump/vm.json

```diff
@@ -0,0 +1,92 @@
+{
+        "vm": [
+                {
+                        "name": "vm-backend-0-3",
+                        "hypervisor": "hypervisor-0-1",
+                        "memory": 2048,
+                        "cpu": 1,
+                        "disks": [
+                                {
+                                        "disk_name": "sda",
+                                        "disk_type": "disk",
```

Comment:
Is it possible to add a test case that covers this conditional `disk_type == 'image':`?

Reply:
For the load command I do that in `test_load_vm_storage`, unlike the dump tests, it uses a pytest fixture to create image type files in a temp file and add it from a virtual machine vs a fixed json file. After adding the images, I dump the VM part of the database, remove the virtual machine hosts, load the VM's again, and make sure a second dump matches the one from before the load.